dashboard/lib/services/script_seed.rb

```diff
@@ -381,6 +391,70 @@ def self.import_lessons_resources(lessons_resources_data, seed_context)
       LessonsResource.joins(:lesson).where('stages.script_id' => seed_context.script.id)
     end
 
+    def self.import_vocabularies(vocabularies_data, seed_context)
+      course_version_id = seed_context.script.get_course_version&.id
+
+      return [] if vocabularies_data.blank?
```

Comment:
Existing `script_json` files don't actually have vocabulary in them so this was needed to avoid the seed process from crashing. We may run into this in the future on other models (eg resources) when we try to launch this seed process on all scripts.